src/Devices/OWSVerificationStateSyncMessage.h

```diff
@@ -0,0 +1,21 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "OWSOutgoingSyncMessage.h"
+#import "OWSRecipientIdentity.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface OWSVerificationStateSyncMessage : OWSOutgoingSyncMessage
+
+- (void)addVerificationState:(OWSVerificationState)verificationState
+                 identityKey:(NSData *)identityKey
+                 recipientId:(NSString *)recipientId;
+
+// Returns the list of recipient ids referenced in this message.
+- (NSArray<NSString *> *)recipientIds;
```

Comment:
I know a lot of people like to do that, but I prefer properties that correspond to state.